Resources/Prototypes/Entities/Mobs/Species/human.yml

```
@@ -4,7 +4,7 @@
   name: Urist McHands
   id: MobHumanBase
   description: A miserable pile of secrets.
-  abstract: true
+  noSpawn: true
```

```suggestion
  abstract: true
```
These guys should stay abstract, nothing should spawn them, they are a helper.

---

they arent spawnable